docs/source/getting_started/benchmarks/imagenette_benchmark.py

```diff
@@ -11,50 +11,52 @@
-| SwaVQueue        |        256 |    200 |              0.845 |   68.8 Min |      6.4 GByte |
-| SMoG             |        256 |    200 |              0.634 |  176.9 Min |     24.2 GByte |
-| TiCo             |        256 |    200 |              0.423 |   43.2 Min |      2.5 GByte |
-| VICReg           |        256 |    200 |              0.733 |   49.2 Min |      3.8 GByte |
-| VICRegL          |        256 |    200 |              0.643 |   48.0 Min |      2.6 GByte |
----------------------------------------------------------------------------------------------
-| BarlowTwins      |        256 |    800 |              0.850 |  279.5 Min |      5.7 GByte |
-| BYOL             |        256 |    800 |              0.887 |  202.7 Min |      4.3 GByte |
-| DCL              |        256 |    800 |              0.864 |  183.7 Min |      3.7 GByte |
-| DCLW             |        256 |    800 |              0.861 |  188.5 Min |      3.7 GByte |
-| DINO (Res18)     |        256 |    800 |              0.887 |  291.6 Min |      8.5 GByte |
-| FastSiam         |        256 |    800 |              0.865 |  280.9 Min |      7.3 GByte |
-| MAE (ViT-S)      |        256 |    800 |              0.620 |  208.2 Min |      4.6 GByte |
-| MSN (ViT-S)      |        256 |    800 |              0.833 |  394.0 Min |     16.3 GByte |
-| Moco             |        256 |    800 |              0.874 |  220.7 Min |      4.2 GByte |
-| NNCLR            |        256 |    800 |              0.885 |  207.1 Min |      3.8 GByte |
-| PMSN (ViT-S)     |        256 |    800 |              0.830 |  401.1 Min |     16.3 GByte |
-| SimCLR           |        256 |    800 |              0.889 |  206.4 Min |      3.7 GByte |
-| SimMIM (ViT-B32) |        256 |    800 |              0.351 |  302.8 Min |     10.5 GByte |
-| SimSiam          |        256 |    800 |              0.871 |  178.2 Min |      3.9 GByte |
-| SwaV             |        256 |    800 |              0.899 |  309.0 Min |      6.4 GByte |
-| SwaVQueue        |        256 |    800 |              0.898 |  300.3 Min |      6.4 GByte |
-| SMoG             |        256 |    800 |              0.782 |  250.2 Min |      2.5 GByte |
-| TiCo             |        256 |    800 |              0.857 |  184.7 Min |      2.5 GByte |
-| VICReg           |        256 |    800 |              0.843 |  192.9 Min |      5.7 GByte |
-| VICRegL          |        256 |    800 |              0.781 |  207.4 Min |      5.7 GByte |
----------------------------------------------------------------------------------------------
+Results (4.5.2023):
+-------------------------------------------------------------------------------------------------
+| Model            | Batch Size | Epochs |  KNN Top1 Val Accuracy |       Time | Peak GPU Usage |
+-------------------------------------------------------------------------------------------------
+| BarlowTwins      |        256 |    200 |                  0.651 |   85.0 Min |      4.0 GByte |
```

Comment:
Big jump in accuracy here but no changes - what happened?

Reply:
I believe we didn't run the 200 epoch tests for a long time. So many changes accumulated (gaussian blur fix, new transforms, ...).  I am not sure what was the reason it increased so much. It is also possible that we previously copied the wrong number by accident.